platform/web/server.go

```diff
@@ -39,6 +41,15 @@ func NewServer(config ServerConfig, router http.Handler) *http.Server {
 	return &server
 }
 
-func NewMonitoringServer(config ServerConfig) *http.Server {
-	return NewServer(config, http.DefaultServeMux)
+func NewMonitoringServer(config ServerConfig, logger *log.Logger, healthzHandler Handler) *http.Server {
+	router := NewRouter(logger, DefaultMiddlewares(logger))
+
+	router.HandleFunc("GET", "/ping", handlePing)
```

Comment:
shouldn't we name it `pingHandler` to be consistent?

Reply:
makes sense